util/cron/common-champs.bash

```diff
@@ -4,33 +4,33 @@
 
 CWD=$(cd $(dirname ${BASH_SOURCE[0]}) ; pwd)
 
-COMMON_DIR=/cray/css/users/chapelu
-if [ ! -d "$COMMON_DIR" ]; then
-  COMMON_DIR=/cy/users/chapelu
-fi
+COMMON_DIR=/cy/users/chapelu
+
+export CHAMPS_COMMON_DIR=$COMMON_DIR/champs-nightly
 
 # All CHAMPS testing is currently on a cray-cs
 module list
 
 source $CWD/common-cray-cs.bash
-source $CWD/common-perf-cray-cs-hdr.bash.bash
+source $CWD/common-perf-cray-cs-hdr.bash
 
+loadCSModule PrgEnv-cray
```

Comment:
Why do we need cce here?

Reply:
This gives us `mpicc`. HDF5 needs MPI. I think it is different than the actual `cce` module, though. I tried that one first with no luck.